### PR TITLE
Fixed example to avoid "Duplicate keys" Vue warnings, make :key unique.

### PR DESCRIPTION
**Pull request by appurist**
e.g. "[Vue warn]: Duplicate keys detected: '3'. This may cause an update error.

### CODE REVIEW

**Comment by chewy94** [comment]
Can you pull in the most recent vuetifyjs.com/master? It should auto-update this PR. Just want to make sure we do not introduce any issues

**Comment by appurist** [comment]
Hmm, I don't understand, it's a commit to a fresh fork from just before the PR. Do you want me to pull to my fork (already done) and then push to my fork again, or repeat the PR? (FYI, I'm not very familiar with PRs from forks, even though that's the usual... I usually commit to a branch in the upstream and PR from the branch.)

Is this related to Travis failing? It _looks_ to me like that broke with @nekosaur's scrolling commits 6 hours ago, no?

**Comment by chewy94** [comment]
Jacek went ahead and fixed what’s I was talking about. And yes, I believe Travis is unrelated to this PR. So we will merge it once we get things on our end resolved

**Comment by KaelWD** [comment]
It's fine, it's because #258 doesn't exist on NPM yet, which is what travis uses.

File: examples/dividers/fullBleed.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <v-icon color="grey lighten-1">star_border</v-icon>
               </v-list-tile-action>
             </v-list-tile>
-            <v-divider v-if="index + 1 &lt; items.length" :key="index"></v-divider>
+            <v-divider v-if="index + 1 &lt; items.length" :key="`divider-${index}`"></v-divider>
```
Comment (KaelWD) [inline examples/dividers/fullBleed.vue]: Can you change this to be `<` instead of `&lt;` too?

Comment (appurist) [reply examples/dividers/fullBleed.vue]: LOL. Sure thing. It's both hilarious and frightening that I didn't even notice that on the same line. ;) (Somehow, I guess I was so focused on the keys.)

Comment (KaelWD) [reply examples/dividers/fullBleed.vue]: ty

**Pull request merged**